lamstare/utils/dptest.py

```diff
@@ -93,7 +94,7 @@ def run_single_head_dptest(exp_path:str, ckpt:int, head:str, test_file:Optional[
             # check if "weighted average of errors" is in the file
             with open(result_file, "r") as f:
                 content = f.read()
-            if True or "weighted average of errors" in content:
+            if "weighted average of errors" in content:
```

Comment:
@anyangml If the test data only contains 1 system, will `dp test` output this section?

Reply:
It won't, but only `output of dp test`. However, this function is no longer used. Now the `dlc` task info is used to determine whether the submitted task has finished, but not watching over output file.